index.js

```diff
@@ -84,6 +84,14 @@ function run(opts, cb) {
 	function log() {
 		opts.log && console.log.apply(this, ['	'].concat([].slice.call(arguments)));
 	}
+
+	function deepValue(obj, path){
+		var nodes = path.split('.');
+		for (var i = 0; i < nodes.length; i++){
+			obj = obj[nodes[i]];
```

Comment:
why do we need to override an input obj ? (dbFrom) :)
create a temp var instead